client/my-sites/checklist/wpcom-checklist/checklist-banner/index.js

```diff
@@ -56,8 +52,7 @@ export class ChecklistBanner extends Component {
 			return false;
 		}
 
-		const sitesNeverShowBanner = store.get( storeKeyForNeverShow );
-		if ( get( sitesNeverShowBanner, String( this.props.siteId ) ) === true ) {
+		if ( getNeverShowBannerStatus( this.props.siteId ) ) {
```

Comment:
This reads from local storage on every render -- not nice. It's a preexisting issue, not introduced by this patch at all, you just put it into spotlight 💡 There's plenty of other places in Calypso that implement the "hide and never show again" logic. Usually by storing a preference bit on backend.

Reply:
Hmm, I could cache the value in memory. I'll take a look to see if I can find all the points in the code where this property is changed, and make them go through the new module.

Reply:
Looks like it isn't set anywhere else. Keeping a module-local cache, then :)